include/af/defines.h

```diff
@@ -496,6 +503,9 @@ namespace af
 #if AF_API_VERSION >= 34
     typedef af_random_engine_type randomEngineType;
 #endif
+#if AF_API_VERSION >= 35
+    typedef af_canny_threshold CannyThreshold;
```

Comment:
We use CamelCase(lower camel case) for the function names in C++ API, Wouldn't enum names in same style be confusing ? Shouldn't we move away to PascalCase(Upper camel case) for enums ?